src/android/patches/makeStringsPatch.js

```diff
@@ -0,0 +1,10 @@
+module.exports = function makeStringsPatch(params) {
+  const patch = Object.keys(params).map(key =>
+    `    <string moduleConfig="true" name="${key}">${params[key]}</string>`
```

Comment:
I think we should escape characters here, as obviously I am not sure how `my-awesome-key` is going to be handled in a Java code.